### PR TITLE
[15.0][MIG] sale_coupon_portal: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
cc @Tecnativa TT42513

@chienandalu please review!

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This should be renamed to `coupon_portal` and change terminology across the XMLs.

File: coupon_portal/i18n/sale_coupon_portal.pot
Comment (chienandalu) [inline coupon_portal/i18n/sale_coupon_portal.pot:1]: You should regenerate translations

**Review by chienandalu — CHANGES_REQUESTED** [review]

**Review by chienandalu — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**